Makefile

```diff
@@ -135,6 +145,9 @@ uninstall-inventory-scripts:
 	done
 	-rmdir -p $(prefix)$(datadir)/mender/inventory
 
+uninstall-inventory-network-scripts:
+	rm -f $(prefix)$(datadir)/mender/inventory/$(INVENTORY_NETWORK_SCRIPTS)
```

Comment:
This won't work for more than one file, see targets above for inspiration on how to remove many.

Reply:
Shite, leftover from the `Geo-script` singular.